Flatpak/org.dolphin_emu.dolphin_emu.metainfo.xml.in

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2016 Jeremy Newton -->
 <component type="desktop-application">
-  <id>org.DolphinEmu.dolphin-emu</id>
+  <id>org.dolphin_emu.dolphin_emu</id>
   <name>Dolphin Emulator</name>
-  <developer id="org.dolphin-emu">
-    <name>Dolphin Emulator Project</name>
-  </developer>
```

Comment:
Did you intentionally remove the `<developer>` tag? According to your own link, it is still an allowed tag in there. (Then again, I didn't see any XSD or DTD, so I don't know if it is potentially just in the wrong location and needs to move to later, or perhaps earlier, in the file.)